.github/workflows/e2e-component-pipeline.yml

```diff
@@ -0,0 +1,180 @@
+---
+name: Component-Pipeline
+on:
+  workflow_dispatch:
+    inputs:
+      e2eTestImage:
+        default: 'litmuschaos/litmus-e2e:ci'
+      goExperimentImage:
+        default: 'litmuschaos/go-runner:ci'
+      newLibImage:
+        default: 'litmuschaos/go-runner:ci'
+      oldLibImage:
+        default: 'litmuschaos/go-runner:latest'        
+      operatorImage:
+        default: 'litmuschaos/chaos-operator:ci'
+      runnerImage:
+        default: 'litmuschaos/chaos-runner:ci'
+      chaosNamespace:
+        default: 'litmus'
+      imagePullPolicy:
+        default: 'Always'
+      experimentImagePullPolicy:
+        default: 'Always'
+      updateWebsite: 
+        default: 'false'
+        
```

Comment:
We can also provide env variables here at the workflow level. It will help in maintaining same envs for different events in the same workflow.

Reply:
This is a useful one, we can use it when we have more than one even (say cornjob).